src/SFA.DAS.RequestApprenticeTraining.Infrastructure/Configuration/FunctionsOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.RequestApprenticeTraining.Infrastructure.Configuration
+{
+    public class FunctionsOptions
+    {
+        public string SampleTimerSchedule { get; set; }
```

Comment:
The SampleTimerFunction can be deleted now

Reply:
removed